application/org.openjdk.jmc.flightrecorder.ui/src/main/java/org/openjdk/jmc/flightrecorder/ui/common/DurationPercentileTable.java

```diff
@@ -85,15 +85,10 @@
 	public static final String TABLE_NAME = "DurationPercentileTable"; //$NON-NLS-1$
 	private static final String COL_ID_PERCENTILE = TABLE_NAME + ".percentile"; //$NON-NLS-1$
 
-	private static final IQuantity[] PERCENTILES = {
-			UnitLookup.NUMBER_UNITY.quantity(0.0),
-			UnitLookup.NUMBER_UNITY.quantity(90.0),
-			UnitLookup.NUMBER_UNITY.quantity(99.0),
-			UnitLookup.NUMBER_UNITY.quantity(99.9),
-			UnitLookup.NUMBER_UNITY.quantity(99.99),
-			UnitLookup.NUMBER_UNITY.quantity(99.999),
-			UnitLookup.NUMBER_UNITY.quantity(100.0),
-	};
+	private static final IQuantity[] PERCENTILES = {UnitLookup.NUMBER_UNITY.quantity(0.0),
+			UnitLookup.NUMBER_UNITY.quantity(90.0), UnitLookup.NUMBER_UNITY.quantity(99.0),
+			UnitLookup.NUMBER_UNITY.quantity(99.9), UnitLookup.NUMBER_UNITY.quantity(99.99),
+			UnitLookup.NUMBER_UNITY.quantity(99.999), UnitLookup.NUMBER_UNITY.quantity(100.0),};
```

Comment:
Do we want to consider allowing longer list literals to all have entries on separate lines?

Reply:
For places where we really want finer control, we can use 
// @formatter:off
...
// @formatter:on

Reply:
Added an example.